Formula/ccfits

```diff
@@ -0,0 +1,23 @@
+class Ccfits < Formula
+  desc "c++ fits api"
+  homepage "https://heasarc.gsfc.nasa.gov/fitsio/CCfits/"
+  url "https://heasarc.gsfc.nasa.gov/fitsio/CCfits/CCfits-2.5.tar.gz"
+  sha256 "938ecd25239e65f519b8d2b50702416edc723de5f0a5387cceea8c4004a44740"
+
+  bottle do
```

Comment:
New formula shouldn't have a bottle block. The CI will add this on merge.

Reply:
bottle deleted